spec/controllers/playlists_controller_spec.rb

```diff
@@ -146,6 +146,53 @@
     end
   end
 
+  describe "update" do
+    let(:user) { users(:jamie_kiesl) }
+
+    before do
+      login(user)
+    end
+
+    it "should update published_at" do
+      expect(playlists(:jamie_kiesl_loves).published_at).to be_nil
+      expect(playlists(:jamie_kiesl_loves).private).to be(true)
+
+      put :update, params: {
+        id: playlists(:jamie_kiesl_loves).id,
+        permalink: 'jamie-loves',
+        user_id: user.login,
+        playlist: {
+          private: false
+        }}
+      expect(playlists(:jamie_kiesl_loves).reload.published_at).not_to be_nil
```

Comment:
Setting `published_at` should only work if the playlist has 2 tracks. 

Can you add a test that < 2 tracks does not set published_at (it's not an option in the UI, for example).

Reply:
Good idea. Will add a spec.